backend/app/routes/dummy/control.js

```diff
@@ -0,0 +1,12 @@
+const service = require('./emailservice');
+const router = require('express').Router({ mergeParams: true });
+router.post('/', email);
+
+function email(req, res, next) {
+    console.log(req.body.email);
+    service.sendVerificationEmail(req.body.email)
```

Comment:
Please use await-to-js here

Reply:
What is the use of another function here? You can directly call sendMail helper function

Reply:
I've created a separate function so that the Email service function does not include the json object and unnecessarily creating confusion for a new person looking at it.